doc/htmldoc/installation/index.rst

```diff
@@ -3,40 +3,63 @@
 Install NEST
 ============
 
-.. grid:: 1 1 2 2
 
-   .. grid-item-card::  |user| Install pre-built NEST package
-       :class-title: sd-d-flex-row sd-align-minor-center
+.. grid:: 2
+   :gutter: 1
 
-       I'm a user who wants to :ref:`install NEST on my computer <user_install>`
+   .. grid-item-card:: |user|  Install a pre-built NEST package using
+      :class-title: sd-d-flex-row sd-align-minor-center
+      :columns: 4
 
+      * :ref:`Docker (cross-platform) <docker>`
+        (Includes NESTML)
+      * :ref:`conda-forge (Linux/macOS) <conda_forge_install>`
+      * :ref:`Ubuntu PPA (Linux) <ubuntu_install>`
+      * :ref:`Homebrew (macOS) <macos_install>`
+      * :ref:`Options for Windows users <windows_install>`
 
-   .. grid-item-card:: |teacher| Install NEST for a class or workshop
-       :class-title: sd-d-flex-row sd-align-minor-center
 
-       I'm a lecturer who wants to :ref:`use NEST to teach <lecturer>`
+   .. grid-item-card:: |dev| Source install
+      :class-title: sd-d-flex-row sd-align-minor-center
+      :columns: 4
 
+      If you want to do development with NEST
 
-.. grid:: 1 1 2 2
+      * :ref:`dev_install`
 
-    .. grid-item-card:: |admin| Install NEST for supercomputers and clusters
-       :class-title: sd-d-flex-row sd-align-minor-center
+.. grid:: 2
+   :gutter: 1
 
-       I'm an admin or user who wants to :ref:`run NEST on HPC <admin_install>`
+   .. grid-item-card:: |desktop|
+      :class-title: sd-d-flex-row sd-align-minor-center
+      :columns: 4
 
-    .. grid-item-card:: |dev| Install NEST from source
-       :class-title: sd-d-flex-row sd-align-minor-center
+      A graphical user interface, ideal for learning and teaching concepts
+      regarding neural networks in classrooms and workshops.
 
-       I'm a developer who wants to :ref:`do development in NEST <dev_install>`
+      :doc:`NEST Desktop documentation <desktop:index>` :octicon:`link-external`
```

Comment:
I would suggest to use bullet points in all the cards for consistency.

Reply:
Ok I updated the text.

Reply:
@clinssen what do you think now?